spec/feature_flagger/control_spec.rb

```diff
@@ -19,19 +20,19 @@ module FeatureFlagger
         it { expect(result).to be_falsey }
 
         context 'and a feature is release to all' do
-          before { storage.add(FeatureFlagger::Control::RELEASED_FEATURES, key) }
+          before { storage.add(FeatureFlagger::Control::RELEASED_FEATURES, resource_name, key) }
```

Comment:
We should be using `control.release` in all control tests.

Reply:
make sense, done